PeerioTechnologies/peerio-desktop
# fix: non-Avatar ContactProfile popups

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8174/desktop-no-popup-when-clicking-on-profile

#### Testing instructions  
ContactProfile popups that are not attached to Avatars were broken after PeerUI extraction. Check that these work:

* `@` mentions in chat
* user list in message sidebar (click on "+4" or whatever when there are a lot of read receipts)
* user list in channel sidebar
* in Files, file owner and folder owner

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
